css/properties/text-rendering.json

```diff
@@ -52,8 +52,11 @@
               "version_added": "4.2"
             },
             "samsunginternet_android": {
-              "version_added": false,
-              "notes": "This property is only supported on Windows and Linux. Samsung Internet is not on Windows or Linux."
+              "version_added": "1.0",
+              "notes": [
+                "This property is only supported on Windows and Linux.",
```

Comment:
The old notes suggest that this shouldn't be listed as supported. That chrome_android is listed as supported is pretty weird too. Can you test whether it's actually supported, and update notes if so?

Reply:
Whoops that was a git mistake on my part that should've been reverted, thanks for the spot!